src/components/ActionMenu/components/SecondaryAction/SecondaryAction.tsx

```diff
@@ -25,7 +26,13 @@ export function SecondaryAction({
   }, [getOffsetWidth]);
 
   return (
-    <span className={styles.SecondaryAction} ref={secondaryActionsRef}>
+    <span
+      className={classNames(
+        styles.SecondaryAction,
+        rest?.destructive && styles.destructive,
```

Comment:
Let's add `destructive` to the deconstruction.

```suggestion
        destructive && styles.destructive,
```